packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -212,17 +219,29 @@ export class ModelScene extends Scene {
    * Sets the point in model coordinates the model should orbit/pivot around.
    */
   setTarget(modelX: number, modelY: number, modelZ: number) {
-    this.model.position.set(-modelX, -modelY, -modelZ);
-    this.isDirty = true;
+    this.goalTarget.set(-modelX, -modelY, -modelZ);
   }
 
-  /**
-   * The AR scene is anchored at the center of the lower, forward edge of the
-   * bounding box.
-   */
-  setARTarget() {
-    const {min, max} = this.model.boundingBox;
-    this.model.position.set(-(min.x + max.x) / 2, -min.y, -max.z);
+  getTarget(): Vector3 {
```

Comment:
All of these new public methods want crave docs 📝

Reply:
Done.